### ISSUE TITLE
Segmented Control

### ISSUE THREAD

**Issue by cathystoli** [chanzuckerberg/sci-components#700]
**Issue:** 
1. SR reads 'toggle button' which does not accurately reflect this component.

2. Does not have focus control indicator.

3. No tooltip being shown when using keyboard navigation.

**Solution:**
1. Is there a better term for the SR to read outloud for this type of component?

2. Add focus control indicators.

3. Tooltip should show up when using keyboard to navigate the component.

**Comment by TheMooseman** [comment]
This appears to be an issue with MUI that I can't resolve, relevant issue below
https://github.com/mui/material-ui/issues/17203

**Comment by masoudmanson** [comment]
New designs needed.

@clarsen-czi
